app/code/community/Bolt/Boltpay/Model/Quote.php

```diff
@@ -175,24 +175,33 @@ function ($item) use ($quote, &$calculatedTotal, $boltHelper) {
             /////////////////////////////////////////////////////////////////////////////////////////
         } else {
 
-            // Billing / shipping address fields that are required when the address data is sent to Bolt.
-            $requiredAddressFields = array(
+            $billingAddress  = $quote->getBillingAddress();
+            $shippingAddress = $quote->getShippingAddress();
+
+            /*////////////////////////////////////////////////////////////////////////////
+               -------------------------------------
+                Billing / shipping address fields that are required
```

Comment:
what is the purpose of this comment?

Reply:
As you can see from the previous version, this was once an array and the commit was called `removed unnecessary array and left as comment`

https://github.com/BoltApp/bolt-magento1/pull/209/commits/4b22bd9f8d1d1a3b1e7d6e6b3f7a46fb67cc8ff8

My original thoughts was to remove it entirely, however, it does provide some useful information to a developer, so, I believe that it adds some sort of long-term benefit and does not do much harm.  Certainly, it can be removed.